### ISSUE TITLE
support a "preferred email address"

### ISSUE THREAD

**Issue by lloyd** [mozilla/persona#491]
It should be possible for a site to recommend an email address that the user use.  One specific use case where this is important is a web app that's been purchased, whereby if the user uses any other email address than the one in the "application receipt", the application won't work.

_how_ to emphasize the "preferred" email address needs UX feedback.  What the API looks like is open for discussion, I suggest options to the getVerifiedEmail call (as @michaelrhanson and @benadida have talked about).

 One issue that needs to be resolved is precedence when the information provided by the site conflicts with the user's previous behavior (namely, issue #1).  I believe that the user's previous behavior should over-ride the preference of the website.

**Comment by lloyd** [comment]
requiredEmail feature has been added.  we'll wait on any preferred email feature and open a new issue if it becomes clear we need such a thing.
